cmd/exo/cmd/privnet_associate.go

```diff
@@ -21,26 +24,47 @@ var privnetAssociateCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
-
-		for _, vm := range args[1:] {
-			resp, err := associatePrivNet(network, vm)
+		table := table.NewTable(os.Stdout)
+		table.SetHeader([]string{"Nic ID", "Virtual Machine ID", "IP Address"})
```

Comment:
Keep in mind the CLI is an alternative to the Portal and not an alternative to the API.

![screenshot from 2018-10-05 09-12-26](https://user-images.githubusercontent.com/1388/46521105-bf8e2b00-c87e-11e8-88ac-982b19dbc1e3.png)

Here we have to show the association of a VM (with its name) to a Network (with its name) with a DHCP assigned IP or not. The NIC identifier is an implementation details we can ignore.